remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 3
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+[Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to remediate unhealthy nodes. The tools are a set of well-known [upstream fence agents (FA)](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments, to fence a node using a reboot action. Moreover, FAR can be used by the [Node HealthCheck Operator](https://github.com/medik8s/node-healthcheck-operator#readme) (or any other operator that supports the external remediation API) as an external remediator.
+
+FAR is recommended when a node becomes unhealthy, and we want to completely isolate the node from a cluster, since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes). The operator is available in the Kuberenets community, [OperatorHub.io](https://operatorhub.io/operator/fence-agents-remediation), and can be installed manually as mentioned in [FAR docuementaion](https://github.com/medik8s/fence-agents-remediation#installation)
+
+As opposed to other remediation systems, FAR requires credentials/access to a management interface (e.g., Intelligent Platform Management Interface ([IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface)), etc.) or an API to reboot a node, but it has many advantages.
+
+### Advantages
+
+* Robustness - FAR can remediate an unhealthy node using a management interface (e.g., IPMI) while still keeping control plane connectivity.
+* Speed - FAR is fast as it adds the FAR NoExecute taint, and deletes workloads to accelerate workload rescheduling, in addition to rebooting the node using a fence agent.
```

Comment:
Workload deletion is critical and not just meant to accelerate (at least as far as statefull workload)

Reply:
I added a paragraph with a comment about that. Let me know if that's good enough for expressing that or if should I rephrase the sentence.